modules/appeals_api/db/migrate/20210420131905_add_metadata_to_notice_of_disagreement.rb

```diff
@@ -0,0 +1,5 @@
+class AddMetadataToNoticeOfDisagreement < ActiveRecord::Migration[6.0]
+  def change
+    add_column :appeals_api_notice_of_disagreements, :metadata, :jsonb
```

Comment:
Is this supposed to be a catchall for any data we 'might' need for NOD? Wouldn't it be better to just add 'boardReviewOption' and subsequent fields as they're needed?

Reply:
The idea was kind of a catchall. Eventually we could put the date from which we want to calculate the 90-day submission window for evidence submission. I'm fine either way. It was a decision Drew and I made in light of the revelation that we are not really going to validate at this point and let all evidence submissions through, but that eventually we will want to validate, but may not be able to hold onto PII for the 90 days.